docs/metrics/prometheus/README.md

```diff
@@ -131,11 +131,17 @@ The list of metrics and its definition are as follows. (NOTE: instance here is o
 | `promhttp_` | all prometheus scrape metrics   |
 
 ### MinIO node specific information
-| name                       | description                                                                    |
-|:---------------------------|:-------------------------------------------------------------------------------|
-| `minio_version_info`       | Current MinIO version with its commit-id                                       |
-| `minio_disks_offline`      | Total number of offline disks on current MinIO instance                        |
-| `minio_disks_total`        | Total number of disks on current MinIO instance                                |
+| name                        | description                                                                    |
+|:----------------------------|:-------------------------------------------------------------------------------|
+| `minio_version_info`        | Current MinIO version with its commit-id                                       |
+| `minio_disks_offline`       | Total number of offline disks on current MinIO instance                        |
+| `minio_disks_total`         | Total number of disks on current MinIO instance                                |
+| `minio_nodes_online`        | Total number of MinIO nodes online                                             |
+| `minio_nodes_offline`       | Total number of MinIO nodes offline                                            |
+| `minio_capacity_total`      | Total capacity available in the cluster (raw)                                  |
+| `minio_capacity_free`       | Total free capacity in the cluster (raw)                                       |
+| `minio_capacity_usable`     | Total capacity usable in the cluster (post erasure coding)                     |
+| `minio_capacity_usable_free`| Total free capacity usable in the cluster (post erasure coding)                | 
```

Comment:
Will discuss this. This is based on the list of missing metrics that was created.

Reply:
IMO this will just become more and more unreliable as we introduce more flexible storage classes.

It will give us more headache to support this than it is worth.

Reply:
since storage classes can be easily changed this value can flip and flip from a usage point of view.

Reply:
Can we rename it as estimated? Or improve the fidelity of it?
We report raw capacity that is free, it would good to give a guidance (not guarantee) for how much more storage can be ingested. It would be foolish for any admin to try to fill up a cluster beyond 80% before adding more capacity.